src/gui/optionsdialog.cpp

```diff
@@ -1057,14 +1057,26 @@ void OptionsDialog::loadOptions()
 #else
     m_ui->autoRunConsole->hide();
 #endif
+
     intValue = pref->getActionOnDblClOnTorrentDl();
     if (intValue >= m_ui->actionTorrentDlOnDblClBox->count())
```

Comment:
Seems the removed functions should be merged into here:
```c++
if ((intValue < 0) || (intValue >= m_ui->actionTorrentDlOnDblClBox->count()))
```

Reply:
All done.